### DISCUSSION TITLE
How to get a snapshot at boot AND hourly

### DISCUSSION THREAD

**Discussion by LightningRhino** [jimsalterjrs/sanoid#875] [Q&A] [answered]
Hi,
this may be a dumb question, but I would like my system (NixOS) to not only have hourly snapshots (and dailys).
How can I achieve to get a snapshot at boot? Sometimes the system is up not longer than 30-45 minutes and I would end up without any snapshot at all.
Is there a way to tell sanoid to create (and purge) now?

**Comment by phreaker0** [comment, marked as answer]
no, but you can add a systemd boot up task which takes a snapshot in the same format like sanoid does at boot. sanoid will purge it like the others according to policy.
